### ISSUE TITLE
Connect to Yandex Cloud PostgreSQL

### ISSUE THREAD

**Issue by Yuriy9728** [Yuriy9728/ca-web-api#113]
Connect to cloud postgres from backend:

- [x] without SSL
- [ ]  with SSL

**Comment by maximastashkin** [comment]
Nothing works for us with SSL jdbc too

**Comment by Yuriy9728** [comment]
@maximastashkin
I used the following [guide](https://cloud.yandex.ru/docs/managed-postgresql/operations/connect). For unknown reasons, I can connect from outside Yandex infrastructure without using SSL. To connect with SSL, I allowed public access for database host, set SSL mode to "verify full" when connecting and installed certificate (as in manual link above). **BUT** SSL is still not used. I checked it with following SQL:
```
SELECT datname,usename, ssl, client_addr 
  FROM pg_stat_ssl
  JOIN pg_stat_activity
    ON pg_stat_ssl.pid = pg_stat_activity.pid;
```
Result:
![image](https://user-images.githubusercontent.com/53396592/187273294-7c0cd7e9-2a8c-46bf-9f09-278be2237ce6.png)
Two of these connections: this application and pgAdmin. In their connections I used SSL mode "verify full".
So everything works same, except that it requires a certificate for verification in client side because of SSL mode.

I came to conclusion that by making host public it doesn't oblige to use SSL-only connection.

If you use Yandex Compute Cloud, Cloud Functions or Serverless containers, then there is no need for SSL.
